ddl-scripts/db2/db2-pim-schema.sql

```diff
@@ -0,0 +1,80 @@
+create sequence MIG_REP_ID_SEQ start with 1 increment by 50;
+create sequence MIGRATION_ID_SEQ start with 1 increment by 50;
+create sequence PLAN_ID_SEQ start with 1 increment by 50;
+
+create table migration_report_logs (
+   report_id bigint not null,
+    log clob(255)
+);
+
+create table migration_reports (
+   id bigint not null,
+    end_date timestamp,
+    migration_id bigint,
+    process_instance_id bigint,
+    start_date timestamp,
+    success smallint,
+    primary key (id)
+);
+
+create table migrations (
+   id bigint not null,
+    cancelled_at timestamp,
+    created_at timestamp,
+    callback_url varchar(255) for bit data,
+    scheduled_start_time timestamp,
+    execution_type integer,
+    kieServerId varchar(255),
+    plan_id bigint,
+    requester varchar(255),
+    error_message clob(255),
+    finished_at timestamp,
+    started_at timestamp,
+    status integer,
+    primary key (id)
+);
+
+create table plan_mappings (
+   plan_id bigint not null,
+    target varchar(255),
+    source varchar(255) not null,
+    primary key (plan_id, source)
+);
+
+create table plans (
+   id bigint not null,
+    description varchar(255),
+    name varchar(255),
+    source_container_id varchar(255),
+    source_process_id varchar(255),
+    target_container_id varchar(255),
+    target_process_id varchar(255),
+    primary key (id)
+);
+
+create table process_instance_ids (
+   migration_definition_id bigint not null,
+    processInstanceIds bigint
+);
+
+create index IDXgiy3fyawbd9nt7mymgd9qd61h on migration_reports (migration_id);
```

Comment:
Can we use a more descriptive name for this index?

Reply:
yes, thanks. Done